www/src/components/package-readme.js

```diff
@@ -51,6 +52,26 @@ const PackageReadMe = props => {
         )}
       </div>
 
+      <div
```

Comment:
Don't you just need to perform similar logic here and check whether `githubUrl` contains gatsbyjs/gatsby?

Reply:
Done